plugin.video.drnu/addon.xml

```diff
@@ -0,0 +1,39 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<addon id="plugin.video.drnu" version="5.0.13" name="DR TV" provider-name="TermeHansen">
+    <requires>
+        <import addon="xbmc.python" version="2.26.0"/>
+        <import addon="script.module.requests" version="2.22.0" />
+        <import addon="script.module.requests-cache" version="0.4.13"/>
+        <import addon="script.module.buggalo" version="1.1.2"/>
+    </requires>
+    <extension point="xbmc.python.pluginsource" library="default.py">
+        <provides>video</provides>
+    </extension>
+    <extension point="xbmc.addon.metadata">
+        <summary lang="en_GB">Watch live and archived TV from DR (Danish Broadcasting Corporation)</summary>
+        <summary lang="da_DK">Se live og arkiveret TV fra DR</summary>
+        <description lang="en_GB">All DR's live TV channels and archived programs as seen on dr.dk/tv.[CR][CR]If you have comments or suggestions for this addon, please feel free to participate in the KODI official forums</description>
+        <description lang="da_DK">Alle DR's live TV kanaler samt alle de programmer du finder på dr.dk/tv.[CR][CR]Har du kommentarer, ris eller ros til denne addon er du velkommen til at deltage i debatten på KODI forummet</description>
+        <disclaimer lang="en_GB">Some parts of this addon may not be legal in your country of residence - please check with your local laws.</disclaimer>
+        <disclaimer lang="da_DK">Nogle dele af denne addon er muligvis ikke lovlig i dit land - kontroller venligst dine lokale love.</disclaimer>
+        <license>GPL-2.0-only</license>
+        <platform>all</platform>
+        <language>da</language>
+        <email>termopeten@gmail.com</email>
+        <website>https://www.dr.dk/</website>
+        <source>https://github.com/xbmc-danish-addons/plugin.video.drnu</source>
```

Comment:
This change is still missing, please correctly identify the upstream location. This is your repo not the old one

Reply:
Tommy have given me and other developers membership to the repo, so we can continue the maintenance and development of the addon. This update 5.0.13 you can see here https://github.com/xbmc-danish-addons/plugin.video.drnu/releases
I could not fork the repo-plugins into github.com/xbmc-danish-addons as I am not the owner, so that is why this PR is coming from my personal github with a fork of repo-plugins (if that is the confusion?)

Reply:
That's fine, nice to know. As long as this is the upstream location of the code, it's fine.